internal/pkg/rpaas/k8s.go

```diff
@@ -206,36 +205,54 @@ func (m *k8sRpaasManager) debugPodWithContainerStatus(ctx context.Context, args
 	return instance, debugContainerName, status, nil
 }
 
-func (m *k8sRpaasManager) generateDebugContainer(ctx context.Context, args *DebugArgs, instance *v1alpha1.RpaasInstance) (string, error) {
+func removeCertVolumeMounts(volumeMounts []corev1.VolumeMount) []corev1.VolumeMount {
+	var result []corev1.VolumeMount
+	for _, vm := range volumeMounts {
+		if !strings.Contains(vm.MountPath, "certs") {
```

Comment:
@ravilock could we use a more specific matcher?

like:

strings.HasPrefix(vm.MountPath, "/etc/nginx/certs")